diff -ru ../vespa-onnxruntime-1.4.0.FCS/onnxruntime/test/providers/cpu/controlflow/loop_test.cc ./onnxruntime/test/providers/cpu/controlflow/loop_test.cc
--- ../vespa-onnxruntime-1.4.0.FCS/onnxruntime/test/providers/cpu/controlflow/loop_test.cc	2020-07-15 06:34:37.000000000 +0200
+++ ./onnxruntime/test/providers/cpu/controlflow/loop_test.cc	2021-02-20 20:21:25.921543343 +0100
@@ -13,6 +13,8 @@
 #include "test/util/include/default_providers.h"
 #include "test/framework/test_utils.h"
 
+#include <thread>
+
 using namespace ONNX_NAMESPACE;
 
 namespace onnxruntime {
Only in ./onnxruntime/test/providers/cpu/controlflow: loop_test.cc~
