diff --git a/onnxruntime/core/providers/cpu/sequence/concat_from_sequence.h b/onnxruntime/core/providers/cpu/sequence/concat_from_sequence.h
index b344ac2a6..24eba7ff3 100644
--- a/onnxruntime/core/providers/cpu/sequence/concat_from_sequence.h
+++ b/onnxruntime/core/providers/cpu/sequence/concat_from_sequence.h
@@ -5,7 +5,7 @@
 
 #include "core/common/common.h"
 #include "core/framework/op_kernel.h"
-#include "core/providers/cpu//tensor/concat.h"
+#include "core/providers/cpu/tensor/concat.h"
 
 namespace onnxruntime {
 
diff --git a/winml/test/scenario/cppwinrt/NoisyReluCpu.h b/winml/test/scenario/cppwinrt/NoisyReluCpu.h
index 771554cdb..4f4442119 100644
--- a/winml/test/scenario/cppwinrt/NoisyReluCpu.h
+++ b/winml/test/scenario/cppwinrt/NoisyReluCpu.h
@@ -4,7 +4,7 @@
 #pragma once
 
 #include "core/providers/dml/DmlExecutionProvider/inc/MLOperatorAuthor.h"
-#include "core/common//common.h"
+#include "core/common/common.h"
 
 struct NoisyReluShapeInferrer : winrt::implements<NoisyReluShapeInferrer, IMLOperatorShapeInferrer>
 {
diff --git a/winml/test/scenario/cppwinrt/ReluCpu.h b/winml/test/scenario/cppwinrt/ReluCpu.h
index 44b59a5f1..03005d89f 100644
--- a/winml/test/scenario/cppwinrt/ReluCpu.h
+++ b/winml/test/scenario/cppwinrt/ReluCpu.h
@@ -4,7 +4,7 @@
 #pragma once
 
 #include "core/providers/dml/DmlExecutionProvider/inc/MLOperatorAuthor.h"
-#include "core/common//common.h"
+#include "core/common/common.h"
 
 struct ReluShapeInferrer : winrt::implements<ReluShapeInferrer, IMLOperatorShapeInferrer>
 {
